src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -459,121 +458,6 @@ def ensure_container_insights_for_monitoring(
-                                        "location": "[parameters('workspaceRegion')]",
-                                        "name": "[Concat('ContainerInsights', '(', split(parameters('workspaceResourceId'),'/')[8], ')')]",
-                                        "properties": {
-                                            "workspaceResourceId": "[parameters('workspaceResourceId')]"
-                                        },
-                                        "plan": {
-                                            "name": "[Concat('ContainerInsights', '(', split(parameters('workspaceResourceId'),'/')[8], ')')]",
-                                            "product": "[Concat('OMSGallery/', 'ContainerInsights')]",
-                                            "promotionCode": "",
-                                            "publisher": "Microsoft",
-                                        },
-                                    }
-                                ],
-                            },
-                            "parameters": {},
-                        },
-                    }
-                ],
-            }
-
-            params = {
-                "workspaceResourceId": {"value": workspace_resource_id},
-                "workspaceRegion": {"value": location},
-                "solutionDeploymentName": {"value": solution_deployment_name},
-            }
-
-            deployment_name = "aks-monitoring-{}".format(unix_time_in_millis)
-            # publish the Container Insights solution to the Log Analytics workspace
-            return _invoke_deployment(
-                cmd,
-                resource_group,
-                deployment_name,
-                template,
-                params,
-                validate=False,
-                no_wait=False,
-                subscription_id=subscription_id,)
-
-
-def _is_container_insights_solution_exists(cmd, workspace_resource_id):
-    # extract subscription ID and resource group from workspace_resource_id URL
-    is_exists = False
-    _MAX_RETRY_TIMES = 3
-    parsed = parse_resource_id(workspace_resource_id)
-    subscription_id, resource_group, workspace_name = parsed[
-        "subscription"], parsed["resource_group"], parsed["name"]
-    solution_resource_id = "/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.OperationsManagement/solutions/ContainerInsights({2})".format(
-        subscription_id, resource_group, workspace_name)
-    resources = cf_resources(cmd.cli_ctx, subscription_id)
-    for retry_count in range(0, _MAX_RETRY_TIMES):
-        try:
-            resources.get_by_id(solution_resource_id, '2015-11-01-preview')
-            is_exists = True
-            break
-        except HttpResponseError as ex:
-            if ex.status_code == 404:
-                is_exists = False
-                break
-            if retry_count >= (_MAX_RETRY_TIMES - 1):
-                raise ex
-    return is_exists
 
 
 def _invoke_deployment(
```

Comment:
I suppose after removing the `if not _is_container_insights_solution_exists` code block, `_invoke_deployment` is also not used anymore. Please help confirm and remove it if it's useless.

Reply:
> `_invoke_deployment`

> Yes, this is unused also hence the removed this.